lib/amazonka/amazonka.cabal

```diff
@@ -120,3 +120,22 @@ library
     , transformers          >=0.2
     , unordered-containers  ^>=0.2.14.0
     , uuid                  >=1.2.6     && <1.4
+
+test-suite amazonka-auth-background
+  default-language:   Haskell2010
+  type:               exitcode-stdio-1.0
+  hs-source-dirs:     test
+  main-is:            Main.hs
+  ghc-options:
+    -Wall -fwarn-incomplete-uni-patterns
+    -fwarn-incomplete-record-updates
+  build-depends:
+    , base                  >=4.12 && <5
+    , amazonka              ^>=2.0
+    , amazonka-core         ^>=2.0
+    , tasty                 >=0.10
+    , tasty-hunit           >=0.9
+    , time                  >=1.9
+    , exceptions            ^>=0.10.4
+    , http-client           >=0.5
+    , http-types            >=0.12
```

Comment:
If a dependency is listed in the `library` target, we do not need to repeat its bounds here.